docs/cluster_management.md

```diff
@@ -31,3 +31,25 @@ You can use the kube-version-change utility to convert config files between diff
 $ hack/build-go.sh cmd/kube-version-change
 $ _output/go/bin/kube-version-change -i myPod.v1beta1.yaml -o myPod.v1beta3.yaml
 ```
+
+### Upgrading the kernel on a node
```

Comment:
Does docker daemon use libc?  It seems we cant avoid downtime in the libc case.

How about `Maintenance event on a node`?